esmvaltool/_task.py

```diff
@@ -408,13 +416,10 @@ def _start_diagnostic_script(self, cmd, env):
         try:
             process = subprocess.Popen(
                 cmd,
-                universal_newlines=True,
-                bufsize=1,
                 stdout=subprocess.PIPE,
```

Comment:
I dont get it - why do we need to remove this (and in general why this PR) - all this does is flush all output to text in new lines :confused:

Reply:
See the discussion in #1059 for the reason behind reverting this PR.

Reply:
yeah cheers, looks pretty hairy - check out my comment there, I suspect it's all about the i/o encoding